packages/contracts/deploy/40_finalize-managing-dao/20_register-managing-dao-on-dao-registry.ts

```diff
@@ -25,6 +31,18 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     daoRegistryAddress
   );
 
+  if (
+    await isENSDomainRegistered(
+      `${daoSubdomain}.${daoDomain}`,
+      ENS_ADDRESSES[network.name]
+    )
```

Comment:
1. In this file, we call: 

```
const registerTx = await daoRegistryContract.register(
    managingDAOAddress,
    deployer,
    daoSubdomain
  )
```

This means that `daoSubdomain` actually incudes root domain. For example: `daoSubdomain` can't be `blabla`, it has to be `blabla.dao.eth`.  but on this line,  ``${daoSubdomain}.${daoDomain}`,` you have it concatinated. Are we sure about this, or you should just passing `daoSubdomain` only ?

2. btw, you're using `ENS_ADDRESSES[network.name]` and for the network where `ens `doesn't exist, this would always fail as it would return udnefined and call to undefined contract would fail.

Reply:
3. if you agree with (1), once you fix it, i think it would be good if in the very early steps, we got a check that dao sbudomain is actually a subdomain of `daoDomain`.

Reply:
3. this is fixed

Reply:
1. `daoSubdomain` doesn't include and shouldn't include the root domain. The `ENSSubdomainRegistrar` contract needs to know the subdomain (called label in ENS contracts) and the parent domain (called node).

When we pass `daoSubdomain` as `bla.dao.eth` the registrar would register `bla.dao.eth.dao.eth` and not `bla.dao.eth`. Thus it needs to be without the parent domain

Reply:
1. correct my bad. would be good if you add this in the deployment checklist + add the check in the script that dao sbudomain is actually a subdomain of daoDomain.